opm/bda/cusparseSolverBackend.hpp

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 1993-2015 NVIDIA Corporation.  All rights reserved.
+ *
```

Comment:
Is this copyright disclaimer correct? Does not look like open source.

Reply:
We overlooked this. The header is not correct, it should have read 'This software contains source code provided by NVIDIA Corporation.' because the EULA specifies that we can distribute our derivation, while mentioning that we based it on their samples.
However, digging deeper, it seems that the NVIDIA EULA is not compatible with GPL.